src/p09.rs

```diff
@@ -0,0 +1,87 @@
+use std::{
+    collections::{HashSet, VecDeque},
+    error::Error,
+};
+
+fn get_neighbor_points(map: &[String], pos: (usize, usize)) -> Vec<(usize, usize)> {
```

Comment:
```suggestion
fn get_neighbor_points(map: &[String], pos: Point) -> Vec<Point> {
```
At some point if this tuple pattern keeps repeating you can start using a struct